ds3_java_cli/src/main/java/com/spectralogic/ds3cli/views/csv/DetailedObjectsPhysicalView.java

```diff
@@ -0,0 +1,74 @@
+/*
+ * *****************************************************************************
+ *   Copyright 2014-2016 Spectra Logic Corporation. All Rights Reserved.
+ *   Licensed under the Apache License, Version 2.0 (the "License"). You may not use
+ *   this file except in compliance with the License. A copy of the License is located at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *   or in the "license" file accompanying this file.
+ *   This file is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR
+ *   CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ *   specific language governing permissions and limitations under the License.
+ * ***************************************************************************
+ */
+
+package com.spectralogic.ds3cli.views.csv;
+
+import com.google.common.collect.ImmutableList;
+import com.spectralogic.ds3cli.models.GetDetailedObjectsResult;
+import com.spectralogic.ds3client.models.*;
+import com.spectralogic.ds3client.utils.Guard;
+
+import java.util.ArrayList;
+
+import static com.spectralogic.ds3cli.util.Utils.*;
+
+public class DetailedObjectsPhysicalView extends CsvView<GetDetailedObjectsResult> {
+    private ImmutableList<DetailedS3Object> objects;
+
+    @Override
+    public String render(final GetDetailedObjectsResult obj) {
+        if (obj == null || (obj.getObjIterator() == null) || !obj.getObjIterator().hasNext()) {
+            return "No objects returned";
+        }
+
+        objects = ImmutableList.copyOf(obj.getObjIterator());
+        initTable(ImmutableList.of("Name", "Bucket", "Owner", "Size", "Type", "Creation Date", "Barcode", "State"));
+
+        return renderTable();
+    }
+
+    @Override
+    protected String[][] formatTableContents() {
```

Comment:
Yes, I did "pencils down" because I knew support was waiting. The CSV views are almost exact copies with the CsvView base class instead of the TableView. That could be done much more leanly but not without touching a lot of files. I wanted to make this available and look at cleaner solutions going forward.